lib/commands/updateApplication.ts

```diff
@@ -0,0 +1,41 @@
+import { FetchResult } from "apollo-link";
+import gql from "graphql-tag";
+
+import { fetchManagement } from "../../src/utils/fetchManagement";
+import { ApplicationInput } from "@lib/@types/Application";
```

Comment:
Wait, is this path alias working during your script execution?

Reply:
That's strange, but this one works. But for consistency, I may use relative path I think.